src/Domain/France/City.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\France;
```

Comment:
Je trouve le domaine un peu particulier… Si tu veux spécifier "france", peut-être créer un sous-dossier country ?

Reply:
Disons que déjà ça n'irait pas juste dans "Regulation" car on pourrait modéliser la "ville d'un arrêté" sans que ça soit assis sur la table des "communes de France"

D'où l'idée d'un dossier séparé déjà

Et du coup le dossier "France"... Pour un dossier Domain/Country/France, why not, j'ai rien contre

Reply:
Oui `Domain\Country\France` me parait être un bon compromis.